README.md

```diff
@@ -71,6 +71,181 @@ You must create you ticket branches off the NCIOCPL repo such that secrets are u
 	* `package.json` and `package-lock.json` - you should know what these are.
 	* `README.md` - this document
 
+## Making API Calls
+API endpoint domain has to be set as an initialization parameter for the application in `/src/index.js`.
+
+One API Domain Multiple Endpoints:
+--
+If a call needs to be made to an endpoint for the current weather data like the Open Weather Map API (https://api.openweathermap.org/data/2.5/weather?q=London).
+The "apiEndpoint" in `/src/index.js` can be set to "https://api.openweathermap.org/data/2.5".
+Once the "apiEndpoint" is set, the service name endpoint to get current weather data can be defined in `/src/services/api/endpoints.js` using the "getEndpoints" by adding one for the current weather data to "urls".
+```
```

Comment:
why do we call it urls and not endpoints?

Reply:
Updated